.git-blame-ignore-revs

```diff
@@ -9,3 +9,18 @@
 
 # SOLR-14920 [google java formatter applied]
```

Comment:
```suggestion
# SOLR-14920 [google java formatter applied]
8081b216e6e17f1827ff4d6102cb408c3c599cfe
```
This is for jwt-auth spotless on branch_9_0

Reply:
I don't see that commit on branch_9_0?

`git log --pretty=oneline | grep -F 8081b216e6e17f1827ff4d6102cb408c3c599cf` 

^^^ returns nothing.

I think I covered the two JWT commits here:

```
➜  solr git:(ignore-spotless-blame-9_0) git log --pretty=oneline | grep -F 8a9e314c4d301f85c216b03b0062b3f070d50c47
8a9e314c4d301f85c216b03b0062b3f070d50c47 SOLR-15907 Apply spotless only on JWT classes (#616)
➜  solr git:(ignore-spotless-blame-9_0) git log --pretty=oneline | grep -F 7299cb079e710f5763b90c6c865c0681c5ecf6e1
7299cb079e710f5763b90c6c865c0681c5ecf6e1 SOLR-15907 Spotless
```

they are at the top of the list (line 12 and 13)

Reply:
My bad, I filtered on branch_9x, your two SHAs are already correct.

Reply:
No worries @janhoy appreciate the extra set of eyes :)